### PR TITLE
Migrate to 3.6

### PR DESCRIPTION
**Pull request by priyankarani**

### CODE REVIEW

**Comment by prakashpp** [comment]
@priyankarani build is failing. Also nereid freezes `flask-login` dependency in 3.4, should be unfreezed in this version

**Comment by priyankarani** [comment]
@prakashpp build will pass once email-queue is released.

File: trytond_nereid/trigger.xml [hunk 1]
```diff
@@ -19,7 +19,7 @@
             <field name="on_create" eval="True" />
             <field name="on_write" eval="True" />
             <field name="on_delete" eval="True" />
-            <field name="condition" eval="'True'" />
+            <field name="condition" eval="'true'" />
```
Comment (prakashpp) [inline trytond_nereid/trigger.xml]: are you sure string true?

**Comment by prakashpp** [comment]
LGTM

File: trytond_nereid/trigger.xml (hunk 1 above)
Comment (priyankarani) [inline trytond_nereid/trigger.xml]: yes without that it was raising error

**Pull request merged**